src/rez/pip.py

```diff
@@ -97,7 +100,17 @@ def run_pip_command(command_args, pip_version=None, python_version=None):
     pip_exe, context = find_pip(pip_version, python_version)
     command = [pip_exe] + list(command_args)
 
-    if context is None:
+    if process_output:
```

Comment:
What is this meant for?

Reply:
If it fails because the version doesn't exist, it will return the available versions. Not entirely sure if it's really needed here.

Reply:
This has to do with the ability to list all available versions of a package based on PyPi for the current Python interpreter. Although a bit of a hack as outlined [here](https://stackoverflow.com/questions/4888027/python-and-pip-list-all-versions-of-a-package-thats-available) there is currently no way of retrieving this list. Causing the pip install to fail is intentional in a way to retrieve the available packages.